e-store/e-store/Assets.xcassets/blue.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
```

Comment:
do you still need those color in assets ??

Reply:
yes , any suggestion?

Reply:
I think you will use it by swift right ?

Reply:
as you see you have two options either swift and remove assets or vice versa but not keep both

Reply:
ok i will use swift file